### DISCUSSION TITLE
Looking for some opensource AI/ML projects to contribute to...

### DISCUSSION THREAD

**Discussion by 91mudit** [community/community#146580] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi all 
I am looking for some AI/ML projects to contribute to can be research paper implementation, new ideas etc.
Let me know if you have something in mind or any references.
Thanks

**Comment by MindfulLearner** [comment]
Hey 

i've saw this: 
https://github.com/eugeneyan/open-llms

You can take a look there!
I generally look for openllama!
https://github.com/openlm-research/open_llama

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @91mudit!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
